capstone-ui/src/Components/InteractiveMap/InteractiveMap.jsx

```diff
@@ -3,7 +3,64 @@ import * as topojson from 'topojson-client';
 import usData from '../../data/us.json';
 import "./InteractiveMap.css";
 
-export default function InteractiveMap() {
+export default function InteractiveMap({ raceType }) {
+
+    const fipsStateCodes = {
```

Comment:
I think this would be better as a json file you can read in. It will also help reduce the lines of code and improve visibility of core logic for this file.

Reply:
Yes, I want to move it to either the constants file or a separate json file, was waiting to get everything to main to do it.